src/variational/bijectors.jl

```diff
@@ -0,0 +1,72 @@
+
+# TODO: Move to Bijectors.jl if we find further use for this.
+"""
+    wrap_in_vec_reshape(f, in_size)
+
+Wraps a bijector `f` such that it operates on vectors of length `prod(in_size)` and produces
+a vector of length `prod(Bijectors.output(f, in_size))`.
+"""
+function wrap_in_vec_reshape(f, in_size)
```

Comment:
Since this function is only used once, I suggest we inline it and add comments to explain its behaviour.

Reply:
done